avm/utilities/pipelines/e2eValidation/resourceRemoval/Initialize-DeploymentRemoval.ps1

```diff
@@ -73,6 +73,7 @@ function Initialize-DeploymentRemoval {
         $RemoveFirstSequence = @(
             'Microsoft.Authorization/locks',
             'Microsoft.VirtualMachineImages/imageTemplates', # Must be removed before their MSI & should be removed before its entities permissions are removed
+            'Microsoft.DevOpsInfrastructure/pools' # Must be removed before vnet role assignments and other resources it depends on like a virtual network
```

Comment:
Pipelines should be disabled before merge

Reply:
Interesting. That's a new learning. Tried troubleshooting the order before but figured it would be an issue with the VNET, not the role assignment. Thanks @eriqua. Approved & merge at will